nava/params.py

```diff
@@ -8,11 +9,25 @@
 
 """
 
+class Engine(Enum):
+    """
+    Nava engine class.
+
+    >>> import nava
+    >>> engine = nava.Engine.ALSA
+    """
+
+    AUTO = "auto"
+    WINSOUND = "winsound"
+    ALSA = "alsa"
+    AFPLAY = "afplay"
+
```

Comment:
Now when user requests for an unavailable engine (an engine which is available in the `Engine` enum but not in the user's system) we raise the following error:
```
nava.errors.NavaBaseError: Sound can not play due to some issues.
```
It's better to handle this exception differently somewhere. We may want to handle this in another PR, but I am raising it here just for the record.

Reply:
We should talk about it later. At the moment, all of the engines only work on one OS. In the future, it's possible that we'll have an engine that works on more than one OS. So, I believe that it's not a bad idea to give this option to the user to choose any engine anywhere.